frontend/src/bundles/profile/components/profile-settings/canvas-preview.ts

```diff
@@ -0,0 +1,62 @@
+//@ts-expect-error: unexpected error
```

Comment:
why do we need it?

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/78347072/73c5e9ce-47cf-4c69-8eff-375605fa1212)